### ISSUE TITLE
Atom Opens blank screen & frozen JUST after fresh install of Atom app in my OS X Yosemite

### ISSUE THREAD

**Issue by henry930** [atom/atom#9812]
OSX Version: 10.10.5
Atom version: current release latest

Blank white screen occurs with no error alert. 1 min later a popup says its frozen and asked if I wait or kill process.

I could not use Atom at all. Reinstalled by deleting Atom.app, redownloading Atom.app & install again

If it were caused by corrupt cached files, what cache should I delete?

<img width="1440" alt="2015-11-25 6 41 29" src="https://cloud.githubusercontent.com/assets/3243211/11395133/348535f6-93a4-11e5-981e-f4726d6a820b.png">

<img width="1440" alt="2015-11-25 6 41 38" src="https://cloud.githubusercontent.com/assets/3243211/11395134/36467724-93a4-11e5-9895-9b208d0d7e6e.png">

Besides, I watched over the issue list history to see that the CPU overload problem rooted inside Atom app. What happened? Are the codes rotten already?

**Comment by lee-dohm** [comment]
Can you give the version number of Atom? Since we went to a [release channel system](http://blog.atom.io/2015/10/21/introducing-the-atom-beta-channel.html) there are now *three* "current" or "latest" versions of Atom.

**Comment by henry930** [comment]
Atom Version 1.2.4

**Comment by VansonLeung** [comment]
Me too, any clues ?

**Comment by VansonLeung** [comment]
removing ~/.atom and ~/Library/Preferences/com.github.atom.plist  fixed the startup freeze though for me

**Comment by piepieninja** [comment]
I had a similar problem - and removing the ~/.atom also worked for me

**Comment by innovative1** [comment]
Just hit this on Atom version 1.13.0 running on OSX El Capitan 10.11.6. 

Atom was working mostly fine but I noticed it run away with memory, using 10GB and rising. 
After quitting and restart got the blank screen. Screen stayed blank and could not open files or folders. (I had two windows open from the previous launch, they were both blank). The issue persisted several restarts. Removing `~/.atom` fixed it.

Screenshot of error attached.

![screen shot 2017-01-21 at 3 35 45 am](https://cloud.githubusercontent.com/assets/447871/22173267/42ad91ee-df8b-11e6-91b0-9da7c9f20869.png)

**Comment by 50Wliu** [comment]
> Just hit this

Note that the original error report said that the entire screen was _white_.  Your issue is different; most likely you've entered invalid CSS into the font family setting.  To fix this, open up `~/.atom/config.cson` using a different editor and delete the "fontFamily" line.

**Comment by innovative1** [comment]
> Note that the original error report said that the entire screen was white.
My screen was black because I'm using the dark theme. It's essentially a blank screen. If I was using a light theme my screen would have been white as well.

**Comment by 50Wliu** [comment]
> My screen was black because I'm using the dark theme. If I was using a light theme my screen would have been white as well. It's essentially a blank screen.

Sorry, but this isn't true.  The white in the first picture is [`#fff`](https://github.com/atom/atom/blob/1fef2fa866b0ddd50ebeb1ebea6fc3f793b2a50b/src/main-process/atom-window.coffee#L33), and is what Atom uses when it hasn't loaded any themes yet.  You can trivially reproduce this by reloading the Atom window and noticing it flashes white before returning to your UI theme's background color.

There was a regression in Atom 1.13.0 where if you had _at any time_ entered invalid CSS into your font family setting, it would start erroring in 1.13.0.  You started experiencing this error after restarting Atom because Atom most likely auto-updated when you restarted it.  1.13.1 is out and restores pre-1.13 behavior, so if you update again you should not run into this anymore.

Once again, it has nothing to do with this issue.

**Comment by nelsonic** [comment]
Was this issue _resolved_?

I am seeing it _now_:
![image](https://user-images.githubusercontent.com/194400/30583048-bb93847e-9d1d-11e7-9dc9-c2d86a335f26.png)
(_blank window when opening atom_)
Note: I'm trying to open atom from command line with:
```
atom .
```
(_this **used to** open the project folder for the CWD in the terminal window_)

My Atom version is _apparently_ "_up-to-date_":
![image](https://user-images.githubusercontent.com/194400/30583152-16b2c630-9d1e-11e7-8baf-7fb544457582.png)

Running on Mac OS Sierra v 10.12.6 (16G29) (_also latest_)

**Comment by nelsonic** [comment]
Deleting `rm -rf ~/.atom` + re-install solved it for me. thanks @inn0vative1 👍

**Comment by benwilhelm** [comment]
I had this problem with a *specific project folder*. I was able to fix it by doing the following 

* Open atom from a different project folder. (hereafter referred to as "Folder A")
* Choosing `File > Add Project Folder`, and add the offending project folder ("Folder X")
* Right click on Folder A in the project pane and select `Remove Project Folder`.
* Folder X (the original offending folder) should now be the only folder in your project panel
* Close atom, and re-open from Folder X. Should now be fixed.

**Comment by djdarkbeat** [comment]
@benwilhelm This worked great, thanks for sharing.

**Comment by ShiroKatana** [comment]
I have the same issue and i don't where to find the .atom file everyone is talking about can someone one help please??

**Comment by rsese** [comment]
Your `.atom` directory [1] is by default located in your home directory - on macOS/Linux that's `~/.atom` and on Windows that's `%USERPROFILE%\.atom` (e.g. `C:\Users\username\.atom`).

[1]: https://flight-manual.atom.io/using-atom/sections/basic-customization/#controlling-where-customization-is-stored-to-simplify-your-workflow

**Comment by ShiroKatana** [comment]
@rsese Thanks but the issue is due to the nvidia gpu. I switched to intel hd graphics to run the program and it worked.

**Comment by bbugh** [comment]
The workaround by @benwilhelm worked for me. I suddenly started getting an Atom screen with nothing loaded, it's been working fine until a few minutes ago, and I haven't installed or changed any packages recently. Odd, but it's working again.

Mac OS Sierra, Atom 1.25.0

**Comment by lee-dohm** [comment]
I suspect that most of what people are reporting here are individual problems and not the same issue, especially since the original issue was all-encompassing and the one that @benwilhelm mentions is specific to a given project directory. Because of this, I'm going to close this issue. If people have a reoccurrence of this issue, please open a new issue and fill in the template so that we can get to the bottom of what is causing the problem and fix it permanently.

**Comment by iloving** [comment]
This issue has been closed but I wanted to add my two cents for future people.

I had this issue happen.  Rather than outright deleting the ~/.atom directory, rename it to something else like ~/.atom-bad.  Relaunch atom.  It should start up brand new, as others have stated.

Now, quit atom, delete the newly created .atom directory, and then rename the old .atom-bad directory back to .atom .  I did this as an experiment, and much to my surprise, atom came back up, without losing my plugins and settings.

I have no idea why this happened, and IMO atom should be able to recover gracefully in situations like this instead of just freak out and lock up, but I am happy I didn't have to reconfigure atom all over again.
